ntbs-service/Startup.cs

```diff
@@ -137,6 +138,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             else
             {
                 app.UseStatusCodePagesWithReExecute("/errors/{0}");
+                app.UseExceptionHandler("/errors/500");
```

Comment:
500s count as an exception, not a statuscode page

Reply:
Ah fair, that makes sense - they're not a standard return route!
Looks like we're getting a bit of repetition going between "Test" and "Production" envs. Perhaps this would be a neater structure for these settings:
```
if (env.IsDevelopment) {
// dev specific
} else {
// "regular"
}

if (env.IsProduction()) {
// production specific
}
```

Unless the tests actually operate fine under `app.UseHsts()`, but we'd have to check that!

Reply:
Yep it indeed works witha pp.UseHsts in the tests so I've combined production and test!